stock_lot_filter_available/models/stock_production_lot.py

```diff
@@ -23,7 +23,26 @@ def _search_product_qty(self, operator, value):
         groups = self.env["stock.quant"].read_group(
             domain, ["lot_id", "quantity"], ["lot_id"]
         )
+        involved_lot_ids = [group["lot_id"][0] for group in groups]
+        operator_mapping = {
+            ">": operator.gt,
```

Comment:
You need to do `import operator` at the top.

Reply:
And there will be a conflict on the name, so you should do `import operator as operator_lib` or similar, and use it with `operator_lib.gt`, etc.